PlcHandler.py

```diff
@@ -0,0 +1,92 @@
+# General imports =================================================================================
+import multiprocessing as mp
+from socket import socket, AF_INET, SOCK_STREAM
+from time import sleep
+from br_threading.WorkQCommands import WorkQCmnd, WorkQCmnd_e
+from br_threading.TimerThread import TimerThread
+
+# Constants ========================================================================================
+PLC_IP = "192.168.0.70"
+PLC_PORT = 69
+
+PLC_SOL_OFFSET = 9
+PLC_REQUEST = 2
+
+REQUEST_DELAY = (1.0/50.0) # in seconds
+
+# Class Definitions ===============================================================================
+class PlcHandler():
+    def __init__(self, plc_workq: mp.Queue):
+        PlcHandler.socket = socket(AF_INET, SOCK_STREAM)
+        PlcHandler.socket.connect(
+            (PLC_IP, PLC_PORT)
+        )
+        PlcHandler.plc_workq = plc_workq
+        print("PLC - thread started")
+
+    @staticmethod
+    def send_command(command: bytes) -> None:
+        """
+        Send a command to the PLC.
+
+        Args:
+            command (bytes):
+                The command to send to the PLC.
+        """
+        PlcHandler.socket.send(command)
+
+    @staticmethod
+    def read_response() -> bytes:
+        return PlcHandler.socket.recv(64)
```

Comment:
Review PLC server and decide how you want to read the data

Reply:
it should be updated for receiving 3 arrays instead